src/helpers/numberOfCharacters.js

```diff
@@ -0,0 +1,9 @@
+function numberOfCharacters(title) {
```

Comment:
Ik zou zelf een functienaam kiezen die wat meer past bij wat het doet en/of teruggeeft. Het lijkt nu alsof deze functie het aantal characters gaat teruggeven